### PR TITLE
Add missing update/comment field to optimistic response

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4455

### CODE REVIEW

File: components/expenses/PayExpenseModal.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, host, error }
   const payoutMethodLabel = getPayoutLabel(intl, payoutMethodType);
   const hasBankInfoWithoutWise = payoutMethodType === PayoutMethodType.BANK_ACCOUNT && host.transferwise === null;
   const isScheduling = formik.values.action === 'SCHEDULE_FOR_PAYMENT';
+  console.log(host);
```
Comment (Betree) [inline components/expenses/PayExpenseModal.js:123]:
```suggestion
```

**Review by Betree — COMMENTED** [review]
All good with the fix, let's just remove the `console.log`

File: components/expenses/PayExpenseModal.js (hunk 1 above)
Comment (SudharakaP) [reply components/expenses/PayExpenseModal.js]: Ops, my bad. 🤯

**Comment by SudharakaP** [comment]
@Betree : I'll be merging this after (and if) CI passes if no objections. 😃

**Pull request merged**